### PR TITLE
Add: CW-09 Workload Implementation

### LINKED ISSUE #7: Introduce Contrib.md

### LINKED ISSUE #8: Introduce a document describing promised features

### LINKED ISSUE #19: Allow environmental variables in GUI

### LINKED ISSUE #20: Write README.MD

### LINKED ISSUE #21: Support working-directory independent execution

### LINKED ISSUE #22: Create AUTHORS file and adjust LICENSE.MD accordingly

### LINKED ISSUE #23: Introduce __main__ guard to API's main.py

### PR DESCRIPTION
**Pull request by Sol-Mane**
Closes #7 
Closes #8 
Closes #19
Closes #20 
Closes #21 
Closes #22 
Closes #23

### CODE REVIEW

**Comment by Sol-Mane** [comment]
Closes #2

**Review by Feynman43 — APPROVED** [review]
Agree with introducing GUI as submodule

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+         Petr Svarny   <petr.s@praguecollege.cz>
```
Comment (Aemilivs) [inline AUTHORS:1]: Our lector is a great person, but I doubt we will be able to get evaluated by him if we list him as an author :)

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,13 @@
 MIT License
 
-Copyright (c) 2022 Vadim Emelin
+Copyright (c) 2022 AUTHORS
+         Petr Svarny   <petr.s@praguecollege.cz>
```
Comment (Aemilivs) [inline LICENSE:4]: Same concern as in AUTHORS

**Review by Aemilivs — CHANGES_REQUESTED** [review]
I believe we should exclude Petr from authors of the project, since he is a professor leading the subject this project is purposed for and is supposed to put a mark on this project.

**Comment by Sol-Mane** [comment]
I don't have an opinion on Petr's authorship, but as nobody else voiced his opinion I'll keep it in. I'm open to an MR that fixes that.

**Pull request merged**